### PR TITLE
[faiss v1.7.1] Add get,set verbose for Index

### PR DESCRIPTION
**Pull request by ava57r**
https://github.com/facebookresearch/faiss/pull/1790

### CODE REVIEW

**Comment by Enet4** [comment]
Thank you for your contributions to `faiss-rs` so far, @ava57r. Once these capabilities are given an actual release tag at the Faiss project, please let me know through these PRs, so that they may be reviewed. An eventual release of faiss-rs (0.10.0) is also soft blocked on an upcoming release, as it depends on the changes to CMake in order to build the C API.

**Comment by ava57r** [comment]
I finished.
You can review it

File: src/index/mod.rs [hunk 1]
```diff
@@ -175,6 +175,12 @@ pub trait Index {
 
     /// Remove data vectors represented by IDs.
     fn remove_ids(&mut self, sel: &IdSelector) -> Result<usize>;
+
+    /// Index verbosity level
+    fn verbose(&self) -> bool;
+
+    /// Set Index verbosity level
+    fn set_verbose(&mut self, value: bool);
```
Comment (Enet4) [inline src/index/mod.rs:183]: Please do not forget to add an implementation of these methods to `GpuIndexImpl` as well (see gpu.rs:225).

**Review by Enet4 — CHANGES_REQUESTED** [review]
This does not compile with the `gpu` Cargo feature, but should be easy to fix.

**Review by Enet4 — APPROVED** [review]

**Pull request merged**